docs/guide/features.md

```diff
@@ -88,12 +88,13 @@ But a few libraries haven't transitioned to this new default yet, including [`li
 
 - [TypeScript documentation](https://www.typescriptlang.org/tsconfig#target)
 
-Vite does not transpile TypeScript with the configured `target` value by default, following the same behaviour as `esbuild`.
+Vite ignores the `target` value in the `tsconfig.json`, following the same behavior as `esbuild`.
```

Comment:
Actually `target` is used to derive the default value of `useDefineForClassFields`, but I think it's more easy to understand if we say "ignore" here even if it's a bit of lie.